extensions/include/extensions/extensions.h

```diff
@@ -8,7 +8,7 @@
 
 namespace Extensions
 {
-constexpr const char* availableExtensions[] = {"extensions:texture loader"};
+constexpr const char* availableExtensions[] = {"extensions:texture loader", "extensions:discord rpc"};
```

Comment:
We should actually find a way to ifdef this so we aren't loading this extension on platforms that don't have discord